client/main.lua

```diff
@@ -298,16 +286,15 @@ CreateThread(function()
         if LocalPlayer.state.isLoggedIn then
             sleep = 100
 
-
             local entering = GetVehiclePedIsTryingToEnter(cache.ped)
             local carIsImmune = false
             if entering ~= 0 and not isBlacklistedVehicle(entering) then
                 sleep = 2000
                 local plate = qbx.getVehiclePlate(entering)
 
                 local driver = GetPedInVehicleSeat(entering, -1)
-                for _, veh in ipairs(config.immuneVehicles) do
-                    if GetEntityModel(entering) == joaat(veh) then
+                for _, vehicle in ipairs(config.immuneVehicles) do
+                    if GetEntityModel(entering) == getHash(vehicle) then
```

Comment:
We should just call joaat directly rather than using the wrapper function, as the wrapper function just adds unneeded abstraction.